src/apps/packet_filter/packet_filter.lua

```diff
@@ -354,17 +354,18 @@ local function generateConformFunctionString(rules)
    T"return function(buffer, size)"
    T:indent()
 
-   if rules.state_track then
-      conntrack.define(rules.state_track)
+   if options.state_track then
+      conntrack.define(options.state_track)
       T"if (function(buffer, size)"
       T:indent()
    end
-   
-   if rules.state_check then
-      conntrack.define (rules.state_check)
-      T('if state_pass("', rules.state_check, '", buffer) then return true end')
+
+   if options.state_check then
+      conntrack.define (options.state_check)
+      T('if state_pass("', options.state_check, '", buffer) then return true end')
    end
 
+   local rules = options.rules or {}
```

Comment:
Documentation says that **rules** is required, but it seems to be optional in the code?